README.md

```diff
@@ -1,4 +1,4 @@
-## Go Dependency Manager (gdm) [![Circle CI](https://circleci.com/gh/sparrc/gdm.svg?style=svg)](https://circleci.com/gh/sparrc/gdm)
+# Go Dependency Manager (gdm) [![Circle CI](https://circleci.com/gh/sparrc/gdm.svg?style=svg)](https://circleci.com/gh/sparrc/gdm)
 
 gdm aims to do as little as possible. It does not copy
 dependencies in-repo and does not require that people use `gdm` to build
```

Comment:
`gdm vendor` does copy dependencies in-repo.

Reply:
👍